master/getting-started/kubernetes/index.md

```diff
@@ -1,31 +1,68 @@
 
-## Getting Started
+Use the navigation bar on the left to view information on Calico for Kubernetes,
+or continue reading for recommended guides to get started.
 
-The following guides help you get started with Calico.
+## Installation Guides
 
-Integration guide
-------------------
+There are several ways to install Calico in Kubernetes.
 
-- [Integration Guide]({{site.baseurl}}/{{page.version}}/getting-started/kubernetes/installation/): Discusses adding Calico to an existing cluster, or integrating Calico with your own install scripts.
+#### [Hosted Install Guide (Beta)]({{site.baseurl}}/{{page.version}}/getting-started/kubernetes/installation/hosted)
 
-Quick-start guides
-------------------
-These guides let you get a cluster set up quickly, and walk you through using Calico for networking and network policy.
+Calico's Hosted Install uses Kubernetes Manifests to install Calico for Kubernetes, using Kubernetes.
+Calico manifests ensure the necessary artifacts and configurations are added to each host, and the necessary services are started.
 
-- [CoreOS Vagrant]({{site.baseurl}}/{{page.version}}/getting-started/kubernetes/installation/vagrant/)
-- [CoreOS on GCE]({{site.baseurl}}/{{page.version}}/getting-started/kubernetes/installation/gce)
-- [CoreOS on AWS]({{site.baseurl}}/{{page.version}}/getting-started/kubernetes/installation/aws)
+>Note: Hosted installs are new for Kubernetes v1.4 and Calico's Manifests are still in Beta. It will become
+the supported installation method going forward.
 
-## Requirements
+#### [Manual Install Guide (Recommended)]({{site.baseurl}}/{{page.version}}/getting-started/kubernetes/installation/)
 
-- The kube-proxy must be started in `iptables` proxy mode.  This is the default as of Kubernetes v1.2.0.
+This guide details how to add manually add Calico to an existing, stock Kubernetes cluster.
+
+
+## Quickstart with "Calico-Ready" Clusters
+
+Calico maintains sample cloud-config files which can be used to launch a Kubernetes
+cluster ready for Calico to be installed on.
+
+The following guides walk through launching a CoreOS cluster using these cloud-config scripts on various cloud providers and locally.
+These Kubernetes clusters will not install Calico for users, leaving you to
+follow either of the above installation methods to run Calico.
+
+#### [CoreOS Vagrant]({{site.baseurl}}/{{page.version}}/getting-started/kubernetes/installation/vagrant/)
+
+This guide uses Vagrant with VirtualBox to launch a Calico-ready cluster.
+
+#### [AWS Tutorial]({{site.baseurl}}/{{page.version}}/getting-started/kubernetes/installation/aws)
+
+AWS cluster configurations tend to vary greatly based off the needs of the user.
+Calico's Manual Install Guide (above) serves as a general guide still applicable
+to any AWS cluster.
+
+#### [GCE]({{site.baseurl}}/{{page.version}}/getting-started/kubernetes/installation/gce)
+
+This guide covers launching a cluster on GCE for use with Calico.
+
+## Usage Tutorials
```

Comment:
I'd just call this "Tutorials" so we're not confusing with the "Usage" section of the docs.